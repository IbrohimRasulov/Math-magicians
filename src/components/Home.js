import React from 'react';

const Home = () => {
  return (
    <div className="home">
      <h2>Welcome to our page!</h2>
      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid qui ipsum eius laborum, adipisci earum velit possimus magnam doloribus atque eaque aperiam inventore. Nobis laudantium rerum excepturi error dignissimos in amet autem veniam, eos aperiam modi nostrum sapiente? Quisquam illo atque accusantium sapiente temporibus, vero, porro sunt hic ratione, quasi magni nobis recusandae assumenda expedita cupiditate blanditiis cumque veritatis iusto optio nemo minus! Maiores corporis illo reiciendis quis assumenda doloremque.</p>
      <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ratione placeat laudantium molestiae suscipit alias exercitationem quia saepe et sed aliquid quasi ipsa ab quae rerum molestias deleniti aspernatur ipsum, minima aliquam soluta nisi quis doloribus consectetur sunt! Nisi nesciunt, assumenda mollitia iste eveniet asperiores et quia debitis illum consequatur amet?</p>
    </div>
  );
};

export default Home;
