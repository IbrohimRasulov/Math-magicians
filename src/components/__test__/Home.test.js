import renderer from 'react-test-renderer';
import Home from '../Home';

test('Render Home component', () => {
  const tree = renderer.create(<Home />);
  expect(tree).toMatchSnapshot();
});
