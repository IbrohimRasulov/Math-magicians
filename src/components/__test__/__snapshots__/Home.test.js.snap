// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Home component 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid qui ipsum eius laborum, adipisci earum velit possimus magnam doloribus atque eaque aperiam inventore. Nobis laudantium rerum excepturi error dignissimos in amet autem veniam, eos aperiam modi nostrum sapiente? Quisquam illo atque accusantium sapiente temporibus, vero, porro sunt hic ratione, quasi magni nobis recusandae assumenda expedita cupiditate blanditiis cumque veritatis iusto optio nemo minus! Maiores corporis illo reiciendis quis assumenda doloremque.
  </p>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ratione placeat laudantium molestiae suscipit alias exercitationem quia saepe et sed aliquid quasi ipsa ab quae rerum molestias deleniti aspernatur ipsum, minima aliquam soluta nisi quis doloribus consectetur sunt! Nisi nesciunt, assumenda mollitia iste eveniet asperiores et quia debitis illum consequatur amet? Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor, pariatur? Nesciunt, aperiam accusamus dignissimos repudiandae officiis obcaecati minus voluptatum, eveniet doloribus illo animi fuga ipsa! Suscipit repudiandae magni aliquam quidem? Lorem ipsum dolor sit amet consectetur adipisicing elit. Id impedit cum quos iusto a sequi iure doloribus veritatis, corrupti dolorum. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tenetur deleniti a quis quos eligendi nostrum incidunt aliquid laudantium, quibusdam voluptate. Voluptatem voluptates iure sapiente quod quisquam! Magni eos ex reiciendis.
  </p>
</div>
`;

exports[`Render component 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid qui ipsum eius laborum, adipisci earum velit possimus magnam doloribus atque eaque aperiam inventore. Nobis laudantium rerum excepturi error dignissimos in amet autem veniam, eos aperiam modi nostrum sapiente? Quisquam illo atque accusantium sapiente temporibus, vero, porro sunt hic ratione, quasi magni nobis recusandae assumenda expedita cupiditate blanditiis cumque veritatis iusto optio nemo minus! Maiores corporis illo reiciendis quis assumenda doloremque.
  </p>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ratione placeat laudantium molestiae suscipit alias exercitationem quia saepe et sed aliquid quasi ipsa ab quae rerum molestias deleniti aspernatur ipsum, minima aliquam soluta nisi quis doloribus consectetur sunt! Nisi nesciunt, assumenda mollitia iste eveniet asperiores et quia debitis illum consequatur amet? Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor, pariatur? Nesciunt, aperiam accusamus dignissimos repudiandae officiis obcaecati minus voluptatum, eveniet doloribus illo animi fuga ipsa! Suscipit repudiandae magni aliquam quidem? Lorem ipsum dolor sit amet consectetur adipisicing elit. Id impedit cum quos iusto a sequi iure doloribus veritatis, corrupti dolorum. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tenetur deleniti a quis quos eligendi nostrum incidunt aliquid laudantium, quibusdam voluptate. Voluptatem voluptates iure sapiente quod quisquam! Magni eos ex reiciendis.
  </p>
</div>
`;

exports[`Render components 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid qui ipsum eius laborum, adipisci earum velit possimus magnam doloribus atque eaque aperiam inventore. Nobis laudantium rerum excepturi error dignissimos in amet autem veniam, eos aperiam modi nostrum sapiente? Quisquam illo atque accusantium sapiente temporibus, vero, porro sunt hic ratione, quasi magni nobis recusandae assumenda expedita cupiditate blanditiis cumque veritatis iusto optio nemo minus! Maiores corporis illo reiciendis quis assumenda doloremque.
  </p>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ratione placeat laudantium molestiae suscipit alias exercitationem quia saepe et sed aliquid quasi ipsa ab quae rerum molestias deleniti aspernatur ipsum, minima aliquam soluta nisi quis doloribus consectetur sunt! Nisi nesciunt, assumenda mollitia iste eveniet asperiores et quia debitis illum consequatur amet? Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor, pariatur? Nesciunt, aperiam accusamus dignissimos repudiandae officiis obcaecati minus voluptatum, eveniet doloribus illo animi fuga ipsa! Suscipit repudiandae magni aliquam quidem? Lorem ipsum dolor sit amet consectetur adipisicing elit. Id impedit cum quos iusto a sequi iure doloribus veritatis, corrupti dolorum. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tenetur deleniti a quis quos eligendi nostrum incidunt aliquid laudantium, quibusdam voluptate. Voluptatem voluptates iure sapiente quod quisquam! Magni eos ex reiciendis.
  </p>
</div>
`;
